src/Admin/AdminHelper.php

```diff
@@ -200,7 +200,9 @@ public function appendFormFieldElement(AdminInterface $admin, $subject, $element
         if (\count($toDelete) > 0) {
             $i = 0;
             foreach ($finalForm->get($childFormBuilder->getName()) as $childField) {
-                $childField->get('_delete')->setData(isset($toDelete[$i]) && $toDelete[$i]);
+                if ($childField->has('_delete')) {
```

Comment:
Can you create a constant for this?

Reply:
This, of course, is possible, but are you sure that this is the smallest problem in this code? It seems to me that such improvements will not greatly affect the quality, because this code needs to be completely rewritten.

Reply:
UPD: Fixed